app/components/address_component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AddressComponent < ViewComponent::Base
+  def initialize(address:, raw_location: nil)
+    super
+    @address = address
+    @raw_location = raw_location
+  end
+
+  def formatted_address
+    if @address.present?
+      address_lines = @address.all_address_lines.map(&:strip)
+      return address_lines.join(', <br>').html_safe # rubocop:disable Rails/OutputSafety
```

Comment:
i couldnt work out what it wanted me to do here but feels fine to leave in for now if you cant figure it out either

Reply:
strange. I thought I had resolved in [9b10173](https://github.com/geeksforsocialchange/PlaceCal/pull/2638/commits/9b1017332dcc523fbb76326bca1dec6ad9ba321f) Looks like I forgot to change this one as well; the solution was to use `sanitize` here instead of `html_safe`. Will fix.

Reply:
ugh it might be prettier/rubocop interplay